boat_simulator/nodes/physics_engine/kinematics_data.py

```diff
@@ -0,0 +1,17 @@
+"""This module contains the kinematics data for the boat."""
+
+from dataclasses import dataclass, field
+from numpy.typing import ArrayLike
+import numpy as np
+
+
+@dataclass
+class KinematicsData:
+    cartesian_position: ArrayLike = field(default_factory=lambda: np.zeros(3, dtype=np.float32))
```

Comment:
I think you can just fill in the parameter `default` rather than `default_factory` so you don't need to provide a lambda function.